frontends/web/src/components/guide/guide.tsx

```diff
@@ -61,14 +62,30 @@ interface ProvidedProps {
 
 type Props = ProvidedProps & SharedProps & TranslateProp;
 
+// guide entries are fetched from the translation files. Manual fallback to is implemented in case an
+// array element is undefined in the current language (can be null if not translated in order, for
+// example).
+function getGuideEntries(t, screen): ImplicitEntry[] {
+    const key = 'guide.' + screen;
+    // For now, fallback to the main fallback language, not cascading.
+    const entriesFallback = i18n.getFixedT(i18n.languages[i18n.languages.length - 1])(key, { defaultValue: [] });
+    const entriesCurrentLng = t(key, { defaultValue: [] });
+    return entriesFallback.map((entry, index) => {
+        if (index < entriesCurrentLng.length && entriesCurrentLng[index] !== null) {
+            return entriesCurrentLng[index];
+        }
+        return entry;
+    });
```

Comment:
JavaScript has no `IndexOutOfBoundsException` and thus the following would be conciser and easier to read for me (but I haven't tried it):
```
return entriesFallback.map((entry, index) => entriesCurrentLng[index] || entry);
```
On a different note, it would be more intuitive to me if we would fill missing entries in the current language and not overwrite the entries where available of the fallback language.